docs/README.md

```diff
@@ -188,7 +188,9 @@ Below you may find all available props to configure your search data (be it by u
 | `facetsBehavior` |  `string`        | Defines the behavior filters will have. When set to `dynamic`, it restricts the results according to the filters that user have already selected. If set to `Static`, all filters will continue to be displayed to the user, even is no results exist.                                                                                                                                                  | `Static`           |
 | `skusFilter`           | `enum` | Controls SKUs returned for each product in the query. The less SKUs needed to be returned, the more performant your shelf query will be. Available value options: `FIRST_AVAILABLE` (returns only the first available SKU), `ALL_AVAILABLE` (only returns available SKUs) and `ALL` (returns all product's SKUs).                                                            | `ALL_AVAILABLE` |
 | `simulationBehavior`     | `enum` | Defines whether the search data will be up-to-date (`default`) or fetched using the Cache (`skip`). The last option should be used only if you prefer faster queries over no having the most up-to-date prices or inventory.                                                               | `default` |
-| `installmentCriteria`               | `enum`                 | Controls what price should be shown when there are different installments options for it. Possible values are: `MAX_WITHOUT_INTEREST` (displayes the maximum installment option with no interest attached) or `MAX_WITH_INTEREST` (displayes the maximum installment option whether it has interest attached or not).                                 | `"MAX_WITHOUT_INTEREST"` |
+| `installmentCriteria`               | `enum`                 | Controls what price should be shown when there are different installments options for it. Possible values are: `MAX_WITHOUT_INTEREST` (displayes the maximum installment option with no interest attached) or `MAX_WITH_INTEREST` (displays the maximum installment option whether it has interest attached or not).                                 | `"MAX_WITHOUT_INTEREST"` |
+| `excludedPaymentSystems`               | `string[]`                 | List of payment systems that should not be taken into account to determine the installments option to be returned. This is only taken into account if `installmentCriteria` is set to `MAX_WITHOUT_INTEREST` or `MAX_WITH_INTEREST`.                                  | `undefined` |
+| `includedPaymentSystems`               | `string[]`                 | List of payment systems that should be taken into account to determine the installments option to be returned. This is only taken into account if `installmentCriteria` is set to `MAX_WITHOUT_INTEREST` or `MAX_WITH_INTEREST`. If this is not set, **all** payment systems are included.                                  | `undefined` |
```

Comment:
```suggestion
| `installmentCriteria`               | `enum`                 | Controls which price should be shown when there are different installments options for it. Possible values are: `MAX_WITHOUT_INTEREST` (displays the maximum installment option with no interest attached to it) or `MAX_WITH_INTEREST` (displays the maximum installment option whether it has interest attached to it or not).                                 | `"MAX_WITHOUT_INTEREST"` |
| `excludedPaymentSystems`               | `[string]`                 | List of payment systems that should *not* be taken into account when displaying the installment options to users. Caution: this prop configuration only works in scenarios where the `installmentCriteria` prop was also declared. In case it was not, all available payment systems will be displayed regardless.   | `undefined` |
| `includedPaymentSystems`               | `[string]`                 | List of payment systems that should be taken into account when displaying the installment options to users. Caution: this prop configuration only works in scenarios where the `installmentCriteria` prop was also declared. In case it was not, all available payment systems will be displayed regardless.                                  | `undefined` |
```